Binary files ./misc/seqan_instrumentation/last_revision_copy/sandbox/my_sandbox/apps/tut2/.tut2.cpp.swp and ./sandbox/my_sandbox/apps/tut2/.tut2.cpp.swp differ
diff -u -r -N -x '*.o' -x Thumbs.db -x .DS_Store -x CMakeCache.txt -x misc/seqan_instrumentation/userdata/id.txt -x /data/seqan/svn/seqan-trunk/misc/seqan_instrumentation/userdata/id.txt -x misc/seqan_instrumentation/userdata/ykuccc7dunih46v6_stats.txt -x /data/seqan/svn/seqan-trunk/misc/seqan_instrumentation/userdata/ykuccc7dunih46v6_stats.txt -x .svn -x bin -x build -x util -x misc -x docs -x docs2 -x extras -x core -x misc/seqan_instrumentation/bin -x /data/seqan/svn/seqan-trunk/misc/seqan_instrumentation/bin -x misc/seqan_instrumentation/last_revision_copy -x /data/seqan/svn/seqan-trunk/misc/seqan_instrumentation/last_revision_copy -x misc/seqan_instrumentation/last_revision_copy -x /data/seqan/svn/seqan-trunk/misc/seqan_instrumentation/last_revision_copy -x misc/seqan_instrumentation/userdata -x /data/seqan/svn/seqan-trunk/misc/seqan_instrumentation/userdata -x misc/seqan_instrumentation/userdata -x /data/seqan/svn/seqan-trunk/misc/seqan_instrumentation/userdata ./misc/seqan_instrumentation/last_revision_copy/sandbox/my_sandbox/apps/tut2/tut2.cpp ./sandbox/my_sandbox/apps/tut2/tut2.cpp
--- ./misc/seqan_instrumentation/last_revision_copy/sandbox/my_sandbox/apps/tut2/tut2.cpp	2012-09-04 15:46:41.000000000 +0200
+++ ./sandbox/my_sandbox/apps/tut2/tut2.cpp	2012-09-04 15:47:37.530856588 +0200
@@ -32,22 +32,20 @@
 // Author: Your Name <your.email@example.net>
 // ==========================================================================
 
-#include <seqan/basic.h>
-#include <seqan/sequence.h>
+#include <iostream>
+#include <seqan/sequence.h> 
+#include <seqan/file.h>
 
-// ==========================================================================
-// Classes
-// ==========================================================================
 
-// ==========================================================================
-// Functions
-// ==========================================================================
-
-// --------------------------------------------------------------------------
-// Function main()
-// --------------------------------------------------------------------------
-
-// Program entry point.
+using namespace seqan;
+// Function to print simple alignment between two sequences with the same length
+template <typename TText1, typename TText2>
+void printAlign(TText1 const & genomeFragment, TText2 const & read)
+{
+        std::cout <<  "Alignment " << std::endl;
+        std::cout << "  genome : " << genomeFragment << std::endl;
+        std::cout << "  read   : " << read << std::endl;
+}
 
 int main()
 {
